### PR TITLE
 fix: error when there is no billing address is shopify order

### PR DESCRIPTION
**Pull request by saadsafda**
fix: error when there is no billing address is shopify order

### CODE REVIEW

File: ecommerce_integrations/shopify/order.py [hunk 1]
```diff
@@ -38,9 +38,9 @@ def sync_sales_order(payload, request_id=None):
 		create_shopify_log(status="Invalid", message="Sales order already exists, not synced")
 		return
 	try:
-		shopify_customer = order.get("customer", {})
-		shopify_customer["billing_address"] = order.get("billing_address")
-		shopify_customer["shipping_address"] = order.get("shipping_address")
+		shopify_customer = order.get("customer", {}) if order.get("customer") is not None else {}
```
Comment (ankush) [inline ecommerce_integrations/shopify/order.py:41]:
```suggestion
		shopify_customer = order.get("customer") if order.get("customer") is not None else {}
```

**Comment by mubi64** [comment]
fixes #280

**Comment by ankush** [comment]
@saadsafda send PR to develop branch in future

**Comment by saadsafda** [comment]
fixes #283 <https://github.com/frappe/ecommerce_integrations/pull/283>

**Pull request merged**